src/axios.ts

```diff
@@ -33,7 +33,10 @@ export function makeAxiosInstanceWithRetry(
 ): AxiosInstance {
   return makeAxiosInstance(config, {
     retries,
-    retryCondition: isRetryableError,
+    retryCondition: (error) => {
+      // Timeouts should be retryable
+      return error.code === 'ECONNABORTED' || isRetryableError(error);
```

Comment:
Timeout errors due to `config.timeout` aren't considered retryable by `isRetryableError(Error)`.

Reply:
https://github.com/search?q=org%3Afaros-ai+isRetryableError&type=code

We do use it in other places, but I think it's triggered in combination with a client side timeout. Otherwise we'll just get a 504 and retry.

Reply:
👍